### ISSUE TITLE
yotta installation hangs at:  pip install virtualenv 

### ISSUE THREAD

**Issue by ksulander** [ARMmbed/yotta_windows_installer#20]
I'm trying to install yotta to my win10 laptop by executing the yotta_install_v0.1.4.exe, but it hangs at pip install virtualenv. I'm running the .exe as administrator. Since there is no error message anywhere, I have no clue what goes wrong, any ideas anyone?

**Comment by ksulander** [comment]
I had to follow the Manual Installation instructions, and use command: 'python -m pip install -U yotta' to do the actual yotta installation after having installed python, GCC and ninja separately. For some reason the pip.exe hangs when executed alone as the instructions propose.

**Comment by sg-** [comment]
Keeping open to further investigate

**Comment by BlackstoneEngineering** [comment]
@ksulander 

Were you using the installer to install to the default location 'c:\yotta' or some place else? Also did you have an internet connection at that time and was the win10 install fresh or had it been slightly worn in?

**Comment by ksulander** [comment]
Yes, I was using the installer and location was C:\Users\Kari\Meluta\yotta

Internet connection was there, and Win10 was the latest at that time.

 

               Kari

 

Lähettäjä: Blackstone Engineering [mailto:notifications@github.com] 
Lähetetty: Monday, December 14, 2015 6:43 PM
Vastaanottaja: ARMmbed/yotta_windows_installer <yotta_windows_installer@noreply.github.com>
Kopio: ksulander <kari.sulander@meluta.fi>
Aihe: Re: [yotta_windows_installer] yotta installation hangs at: pip install virtualenv (#20)

 

@ksulander <https://github.com/ksulander>  

Were you using the installer to install to the default location 'c:\yotta' or some place else? Also did you have an internet connection at that time and was the win10 install fresh or had it been slightly worn in?

—
Reply to this email directly or view it on GitHub <https://github.com/ARMmbed/yotta_windows_installer/issues/20#issuecomment-164489730> .



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by BlackstoneEngineering** [comment]
Kari,
Try the latest installer, it should work now. I just tried it on both x64 and x32 win10 installations with no problems. Would like to see if this is still a problem.
